docker-compose.yml

```
@@ -1,9 +1,13 @@
     build: .
-    command: pnpm run dev
+    command: pnpm exec next dev
     restart: unless-stopped
     volumes:
       - .:/app
     ports:
       - '3000:3000'
+    environment:
+      - NODE_ENV=development
+      - CHOKIDAR_USEPOLLING=true
+      - WATCHPACK_POLLING=true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)